decrypt_odrive_file.py

```diff
@@ -71,38 +123,41 @@ def decrypt_file(in_file, out_file, password):
     cipher = Crypto.Cipher.AES.new(key, Crypto.Cipher.AES.MODE_CBC, iv)
     next_chunk = ''
     finished = False
-    
     while not finished:
         chunk, next_chunk = next_chunk, cipher.decrypt(in_file.read(4096 * Crypto.Cipher.AES.block_size))
         if len(next_chunk) == 0:
-                chunk = unpad_pkcs7(chunk)
-                fileHash = chunk[-(calcHash.digest_size):]
-                chunk = chunk[:-(calcHash.digest_size)]
-                finished = True
+            chunk = unpad_pkcs7(chunk)
+            fileHash = chunk[-(calcHash.digest_size):]
+            chunk = chunk[:-(calcHash.digest_size)]
+            finished = True
         out_file.write(chunk)
         calcHash.update(chunk)
```

Comment:
Yeah this is based on V1, which doesn't have encrypt-then-mac encryption authentication and a smaller key (128 vs 256)